homeassistant/components/deconz/config_flow.py

```diff
@@ -97,22 +95,32 @@ async def async_step_user(self, user_input=None):
 
         LOGGER.debug("Discovered deCONZ gateways %s", pformat(self.bridges))
 
-        if len(self.bridges) == 1:
-            return await self.async_step_user(self.bridges[0])
-
-        if len(self.bridges) > 1:
+        if self.bridges:
             hosts = []
 
             for bridge in self.bridges:
                 hosts.append(bridge[CONF_HOST])
 
             return self.async_show_form(
-                step_id="init",
-                data_schema=vol.Schema({vol.Required(CONF_HOST): vol.In(hosts)}),
+                step_id="user",
+                data_schema=vol.Schema(
+                    {
+                        vol.Optional(CONF_HOST): vol.In(hosts),
+                        vol.Optional(CONF_MANUAL_INPUT): bool,
```

Comment:
Feels weird that we have a boolean to pick manual input, but also need to pick a host. Can we make "manual setup" an entry in the hosts?

Reply:
Sure

Reply:
It won't be translatable though